### PR TITLE
test(e2e): add screenshots

### PR DESCRIPTION
**Pull request by louis-bompart**
With that, we should have screenshot artifacts on the tests.

Jest has been quite disappointing here.  you cannot get nice/good information on what the current test is (name, status, etc).

So either I was taking the screenshot every time, or I was adding complexity for the screenshots.
Well, we're not charged for the artifact loading, so I chose the first option.

### CODE REVIEW

**Comment by louis-bompart** [comment]
(For now waiting to see how it's going in the CI 🍿)

File: packages/cli-e2e/utils/browser.ts [hunk 1]
```diff
@@ -50,6 +54,21 @@ export async function getNewBrowser(): Promise<Browser> {
...
 }
 
+export async function captureScreenshots(browser: Browser): Promise<void> {
+  mkdirSync(SCREENSHOTS_PATH, {recursive: true});
+  for (const page of await browser.pages()) {
+    try {
+      await page.screenshot({
+        fullPage: true,
+        type: 'png',
+        path: resolve(SCREENSHOTS_PATH, expect.getState.name),
```
Comment (y-lakhdar) [inline packages/cli-e2e/utils/browser.ts:64]: Should we clean the string before (remove dots for instance)? Maybe it's not important.

**Comment by y-lakhdar** [comment]
Can we also get screenshots after the login flow?

**Review by y-lakhdar — APPROVED** [review]

File: packages/cli-e2e/utils/browser.ts [hunk 2]
```diff
@@ -50,6 +54,25 @@ export async function getNewBrowser(): Promise<Browser> {
   });
 }
 
+export async function captureScreenshots(browser: Browser): Promise<void> {
+  mkdirSync(SCREENSHOTS_PATH, {recursive: true});
```
Comment (olamothe) [inline packages/cli-e2e/utils/browser.ts:58]: Could add a preventive check on "folder does not already exists"

**Review by olamothe — APPROVED** [review]

**Comment by olamothe** [comment]
So, where do we go exactly to see the screenshots ?

**Comment by louis-bompart** [comment]
> So, where do we go exactly to see the screenshots ?
> 
> image
> 
> I'm seeing "no artifacts"
> 🤔

still debugging the CI 😅

**Pull request merged**